docs/docs/optimizing-plugins-with-guessjs.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Optimizing plugins with Guess.js
+---
+
+Pre-loading resources is a great way to improve application performance. However, pre-loading everything on a page can waist bandwidth. This is especially true for mobile phone users who may have limited data and bandwidth. So how do you know which resources to pre-load? Is it just a guessing game? Not anymore.
```

Comment:
I think both spellings are ok, but I'll update it if it needs to be changed. It's probably best to use whatever convention is already in place. I didn't think to double check  before.

Reply:
preload seems common. @LekoArts I am a native English speaker, and it's tricky for me since technology moves fast, and so does language. Language about technology changes pretty fast!